public-ui/src/app/shared/components/pop-up/PopUpReactComponent.tsx

```diff
@@ -0,0 +1,66 @@
+import {
+  AfterViewInit,
+  Component,
+  ElementRef,
+  EventEmitter,
+  Injector,
+  Input,
+  Output,
+  ViewChild,
+  ViewEncapsulation
+} from '@angular/core';
+import * as React from 'react';
+import { FunctionComponent } from 'react';
+import * as ReactDOM from 'react-dom';
+import { BaseReactWrapper } from '../../../data-browser/base-react/base-react.wrapper';
+const containerElementName = 'myReactComponentContainer';
+
+const PopUpReactComponent =
+    (props) => {
+    return <div className='data-statement'>
+            <div className='card'>
+                <div onClick={props.closeClick} className='close'>x</div>
+                   <h2 className='card-title'>{props.title}</h2>
+                   <div className='card-body' dangerouslySetInnerHTML=
```

Comment:
Using `dangerouslySetInnerHTML` should be avoided if possible. Might be better to convert the `HelpTextService` to a constant since it seems to be static content and call it directly here.

Reply:
Added helptext constants in here.